src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/UserAssignedIdentities/RemoveAzureRmUserAssignedIdentityCmdlet.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ManagedServiceIdentity.Common;
+using Microsoft.Azure.Commands.ManagedServiceIdentity.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Resources.Models;
+
+namespace Microsoft.Azure.Commands.ManagedServiceIdentity.UserAssignedIdentities
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmUserAssignedIdentity", SupportsShouldProcess = true)]
+    [OutputType(typeof(bool))]
+    public class RemoveAzureRmUserAssignedIdentityCmdlet : MsiBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = Constants.ResourceGroupAndNameParameterSet,
+            HelpMessage = "The resource group name.")]
+        [ResourceGroupCompleter()]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            Position = 1,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = Constants.ResourceGroupAndNameParameterSet,
+            HelpMessage = "The Identity name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = Constants.InputObjectParameterSet,
+            ValueFromPipeline = true,
+            HelpMessage = "The Identity object.")]
+        [ValidateNotNullOrEmpty]
+        public PsUserAssignedIdentity Identity { get; set; }
```

Comment:
Please rename this to InputObject and add alias to "Identity" (that is what a PowerShell user would expect).